projects/BFDR/BirthRecord_fieldsAndCreateMethods.cs

```diff
@@ -135,7 +138,22 @@ private void CreateAttendant()
             Attendant.Meta.Profile = attendant_profile;
             // Not linked to Composition or inserted in bundle, since this is run before the composition exists.
         }
-    }
+
+        /// <summary>Create MotherHeight.</summary>
+        private void CreateMotherHeight()
```

Comment:
no longer need to have create methods from each observation, instead use the GetObservation and CreateObservationEntry helpers